bin/shodan

```diff
@@ -1070,6 +1077,13 @@ def stream(color, fields, separator, limit, datadir, ports, quiet, timeout, stre
 
     if datadir:
         fout = open_file(datadir, last_time)
+        fout.write('[\n')
```

Comment:
Wrapping the data in an array breaks a lot of existing parsers for the Shodan JSON files and moves away from its streaming model so any change like this would need to be opt-in by the user. And the JSON files can get very large so I would imagine that any processing of the files wouldn't involve loading the entire file into memory.